tools/android_lint/bin/main.dart

```diff
@@ -106,14 +104,13 @@ Future<int> runLint(ArgParser argParser, ArgResults argResults) async {
   if (html) {
     lintArgs.addAll(<String>['--html', argResults['out'] as String]);
   }
-  final String? javaHome = await getJavaHome();
+  final String javahome = getJavaHome(inArgument);
+  print('Using JAVA_HOME=$javahome');
```

Comment:
nit: Is this print meant to be here?

Reply:
it's intended. It helps inform the user of this script that this ENV variable is set. Particularly, relevant in CI.